tests/test_election_types2.py

```diff
@@ -0,0 +1,139 @@
+import votekit.election_types as et
+from votekit.pref_profile import PreferenceProfile
+from votekit.ballot import Ballot
+from votekit.election_state import ElectionState
+from fractions import Fraction
+
+
+def list_equal(list1, list2):
+    if len(list1) == len(list2):
+        if all([l1 == l2 for l1, l2 in zip(list1, list2)]):
+            return True
+    return False
+
+
+def read_profile(file):
+    ballot_pool = []
+    with open(file, "r") as f:
+        for line in f:
+            cands = line.strip().split(",")
+            ballot_pool.append(
+                Ballot(ranking=[set(c) for c in cands], weight=Fraction(1))
+            )
+
+    return PreferenceProfile(ballots=ballot_pool)
+
+
+def equal_ElectionStates(state1, state2):
+    return (
+        list_equal(state1.get_all_winners(), state2.get_all_winners())
+        and list_equal(state1.get_all_eliminated(), state2.get_all_eliminated())
+        and list_equal(state1.get_rankings(), state2.get_rankings())
+    )
+
+
+def _test_Bloc(profile, seats, target_state):
+    bloc_election = et.Bloc(profile=profile, seats=seats)
+    outcome = bloc_election.run_election()
+    return equal_ElectionStates(outcome, target_state)
+
+
+def _test_SNTV(profile, seats, target_state):
+    sntv_election = et.SNTV(profile=profile, seats=seats)
+    outcome = sntv_election.run_election()
+    return equal_ElectionStates(outcome, target_state)
+
+
+def _test_Hybrid(profile, r1_cutoff, seats, target_state):
+    hybrid_election = et.SNTV_STV_Hybrid(
+        profile=profile,
+        r1_cutoff=r1_cutoff,
+        seats=seats,
+        transfer=et.fractional_transfer,
+    )
+    outcome = hybrid_election.run_election()
+    return equal_ElectionStates(outcome, target_state)
+
+
+def test_battery_Bloc():
+    test_profile = read_profile("data/zach_test_ballots.csv")

```

Comment:
one thing to consider: if future code breaks these tests, will a person looking a the test be able to understand the likely culprit and where to look?  I'd consider either adding some comments to explain what zach_test_ballots looks like and what expected results would be and why, or building smaller, more explicit tests that test specific behaviors of the various methods